src/CEOFESABundle/Controller/SessionController.php

```diff
@@ -334,8 +343,12 @@ private function createParticipantForm($of,$module,$moduletype)
     /**
      * Displays a form to edit an existing Session entity.
      *
-     * @Route("/edit/{id}", name="session_edit")
+     * @Route(
+     *     "/edit/{id}",
+     *     name="session_edit"
```

Comment:
CS 
And when there is more than one arguments in an annotation, write them in an explicit manner (with the argument name)